client/src/stories/0-Welcome.stories.js

```diff
@@ -1,13 +1,13 @@
-import React from 'react';
```

Comment:
Do you have prettier code rules. Make sure to run them trough all files. See more on slack